analyzers/Papas.py

```diff
@@ -85,7 +86,7 @@ def process(self, event):
                                    layer=1)
         simparticles = sorted( pfsim_particles,
                                key = lambda ptc: ptc.e(), reverse=True)
-        particles = sorted( self.simulator.particles,
-                            key = lambda ptc: ptc.e(), reverse=True)
+        #energy sorted particles = sorted( self.simulator.particles,
```

Comment:
self.simulator.particles only exists if do_reconstruct is True, and we remove this flag, 
so these commented lines should go away